### ISSUE TITLE
SignatureDoesNotMatch when using PresignedGetObject

### ISSUE THREAD

**Issue by flaviocarmo** [minio/minio-dotnet#123]
Using the following code, got a SignatureDoesNotMatch error using the resulting URL:

```
var client = new MinioClient("s3.amazonaws.com","XXXXXXXXXXXXXXXXXXXX",
"YGUDHGSAGHAUSHKASHKJAHSKHAKjs5dr5ads").WithSSL();

			using (var writer = new StreamWriter("link.txt"))
			{
				writer.Write(client.Api.PresignedGetObject("my.bucket", "prod/temp/file.docx", 1000));
			}
```

Error bellow:

```
<Error>
<Code>SignatureDoesNotMatch</Code>
<Message>
The request signature we calculated does not match the signature you provided. Check your key and signing method.
</Message>
<AWSAccessKeyId>XXXXXXXXXXXXXXXXXXXXX</AWSAccessKeyId>
<StringToSign>
AWS4-HMAC-SHA256 20170308T154749Z 20170308/us-east-1/s3/aws4_request 0204bf34e0144f01793e0b817e843b88a5b079d72e862151a6c5e9a1f095062b
</StringToSign>
```

**Comment by deekoder** [comment]
Fixed. Will be merged soon.

**Comment by harshavardhana** [comment]
Fixed in #124
